client/components/TestPlanVersionsPage/index.jsx

```diff
@@ -534,125 +629,182 @@ const TestPlanVersionsPage = () => {
                             <PhasePill fullWidth={false}>
                                 {testPlanVersion.phase}
                             </PhasePill>
-                            &nbsp;on&nbsp;{getEventDate(testPlanVersion)}
-                        </H2>
-                        <PageUl>
-                            <li>
-                                <FontAwesomeIcon
-                                    icon={faCodeCommit}
-                                    color="#818F98"
-                                    size="xs"
-                                />
-                                <a
-                                    target="_blank"
-                                    rel="noreferrer"
-                                    href={`https://github.com/w3c/aria-at/commit/${testPlanVersion.gitSha}`}
-                                >
-                                    Commit {testPlanVersion.gitSha.substr(0, 7)}
-                                    : {testPlanVersion.gitMessage}
-                                </a>
-                            </li>
-                            <li>
-                                <a
-                                    target="_blank"
-                                    rel="noreferrer"
-                                    href={`/test-review/${testPlanVersion.gitSha}/${testPlanDirectory}`}
+                            &nbsp;on&nbsp;
+                            {getEventDate(testPlanVersion)}
+                        </span>
+                    );
+                })}
+                disclosureContainerView={testPlanVersions.map(
+                    testPlanVersion => {
+                        const vString = `V${convertDateToString(
+                            testPlanVersion.updatedAt,
+                            'YY.MM.DD'
+                        )}`;
```

Comment:
Is this needed after #804? Test plan versions have a `versionString` property now.

Reply:
Got it!